src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -626,6 +626,11 @@ class Settings extends Component {
 
   // Submit selected Settings
   handleSubmit = () => {
+    var x = document.getElementById('snackbar');
```

Comment:
Please don't make use of vanilla JS code. We have a well-placed view lib just for this purpose. Refer to other files where snackbar is being used and try to use a similar code pattern.

Reply:
Ok, would make the changes soon..